_rules/image-button-accessible-name-59796f.md

```diff
@@ -29,7 +29,7 @@ htmlHintIgnore:
 
 ## Applicability
 
-The rule applies to any HTML `input` element with a `type` attribute in the `Image Button` state, that is [included in the accessibility tree][].
+The rule applies to any HTML `input` element with a `type` attribute in the [`Image Button` state](<https://html.spec.whatwg.org/#image-button-state-(type=image)>), that is [included in the accessibility tree][].
```

Comment:
The angle brackets (`<` and `>`) around the link are added by the markdown linter. I assume because of the parenthesis at the end of the URL. So, in short, they're correct and I cannot remove them 😁